hack/run-functests.sh

```diff
@@ -1,36 +1,41 @@
 #!/bin/bash
 set -x
-
-# make sure the test runs with specific go vervsion.
-# install go in <ptp-operator-repo>/bin if not already installed
-GO_VERSION=1.20.4
-REPO_BIN_PATH=$(pwd)/bin
-echo "REPO_BIN_PATH is ${REPO_BIN_PATH}"
-export PATH="${REPO_BIN_PATH}/go/bin:$PATH"
-if go version | grep -q "go${GO_VERSION}"; then
-	echo "Go ${GO_VERSION} is already installed."
+# Set go version
+pattern = "/4.1[0-9]/"
```

Comment:
there should not be blanks around the '=', also I don't think "/" is needed, you can use ^4\.1[0-9]